enterprise/audit/table.go

```diff
@@ -129,22 +129,25 @@ var auditableResourcesTypes = map[any]map[string]Action{
 		"archived":                ActionTrack,
 	},
 	&database.User{}: {
-		"id":                   ActionTrack,
-		"email":                ActionTrack,
-		"username":             ActionTrack,
-		"hashed_password":      ActionSecret, // Do not expose a users hashed password.
-		"created_at":           ActionIgnore, // Never changes.
-		"updated_at":           ActionIgnore, // Changes, but is implicit and not helpful in a diff.
-		"status":               ActionTrack,
-		"rbac_roles":           ActionTrack,
-		"login_type":           ActionTrack,
-		"avatar_url":           ActionIgnore,
-		"last_seen_at":         ActionIgnore,
-		"deleted":              ActionTrack,
-		"quiet_hours_schedule": ActionTrack,
-		"theme_preference":     ActionIgnore,
-		"name":                 ActionTrack,
-		"github_com_user_id":   ActionIgnore,
+		"id":                           ActionTrack,
+		"email":                        ActionTrack,
+		"username":                     ActionTrack,
+		"hashed_password":              ActionSecret, // Do not expose a users hashed password.
+		"created_at":                   ActionIgnore, // Never changes.
+		"updated_at":                   ActionIgnore, // Changes, but is implicit and not helpful in a diff.
+		"status":                       ActionTrack,
+		"rbac_roles":                   ActionTrack,
+		"login_type":                   ActionTrack,
+		"avatar_url":                   ActionIgnore,
+		"last_seen_at":                 ActionIgnore,
+		"deleted":                      ActionTrack,
+		"quiet_hours_schedule":         ActionTrack,
+		"theme_preference":             ActionIgnore,
+		"name":                         ActionTrack,
+		"github_com_user_id":           ActionIgnore,
+		"hashed_one_time_passcode":     ActionSecret, // Do not expose a users one time passcode.
+		"one_time_passcode_expires_at": ActionIgnore,
+		"must_reset_password":          ActionIgnore,
```

Comment:
I think it could be useful to track these?

Reply:
Sure thing, will make that change!